docs/developers/dependencies.md

```diff
@@ -0,0 +1,52 @@
+
+## Dependency Management
+
+Reinventing the wheel takes too long, so dependencies are common. Also, you may want your plugin to hook into another.
```

Comment:
This lead looks incomplete (at least the last sentence comes off as if you wanted to finish writing something but didn't get to it).